### PR TITLE
eslint cleanup and minor additions

### PR DESCRIPTION
**Pull request by rhasselle-oddball**
- Fix eslint issues in generator/app/index.js (automatic)
- Fix eslint issues in generators/form/index.js (automatic)
- Add footerContent
- Add forceShowFormControls to be able to test the form right away

Tested yarn new:app BLANK, SIMPLE, COMPLEX

### CODE REVIEW

File: generators/app/index.js
Comment (rhasselle-oddball) [inline generators/app/index.js:1]: no changes to this file - just eslint

File: generators/form/index.js
Comment (rhasselle-oddball) [inline generators/form/index.js:27]: no changes to this file - just eslint

**Review by jamigibbs — APPROVED** [review]
Thanks for working on this, Robert! I think if you want this in vets-website after it's merged, you'll also want to update the version in `package.json` in this PR and have someone with [NPM publishing](https://www.npmjs.com/package/@department-of-veterans-affairs/generator-vets-website) permission publish it. Then vets-website will need to be updated with that version of the package.

**Review by ndsprinkle — APPROVED** [review]

**Comment by rhasselle-oddball** [comment]
> Thanks for working on this, Robert! I think if you want this in vets-website after it's merged, you'll also want to update the version in `package.json` in this PR and have someone with [NPM publishing](https://www.npmjs.com/package/@department-of-veterans-affairs/generator-vets-website) permission publish it. Then vets-website will need to be updated with that version of the package.

Oh, good to know thanks. I have more changes coming, so maybe we can wait until all those get merged.

**Pull request merged**
